src/plugin/connector/email/category_connector.py

```diff
@@ -0,0 +1,32 @@
+import logging
+import requests
+
+from plugin.connector.base import NHNCloudBaseConnector
+
+_LOGGER = logging.getLogger("cloudforet")
+
+
+class CategoryConnector(NHNCloudBaseConnector):
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+    @staticmethod
+    def list_categories(app_key, secret_key) -> list:
+        categories = []
+        idx = 1
+        while True:
+            response = requests.get(
+                f"https://email.api.nhncloudservice.com/email/v2.1/appKeys/{app_key}/categories?pageNum={idx}",
```

Comment:
It’s late, but I have a question.
In some cases, a version is used in the Request Path among NHN Cloud service APIs.
I would like to suggest managing the service version in settings or external files.

Reply:
If there are many versions, which version should we choose? latest? or any version is OK?

It looks like that we have to manage endpoint url and version each other.

Reply:
@GibeomOh 
Is your intention because variables are more readable than hard-coded string?

Connector depends on a specific version because each api version have different spec. Therefore, it is expected that there will be no case of changing the version.
<img width="265" alt="image" src="https://github.com/cloudforet-io/plugin-nhncloud-inven-collector/assets/55343124/a0681e50-b424-4bd1-b124-b70f5f6b80f1">

Reply:
I use latest version for each resources now.

Reply:
For these kinds of situation, we prepared options field. Actually there is no use-case now. The intention of option field is that
1) When plugin is created
2) User or admin can configure global variables for this plugin like
- custom endpoints
- custom version
- ...

Let's try option field

Reply:
@junho100 
I was considering whether we needed a feature to specify (or change) the version used in the connector. However, if the version differs depending on the connector, it shouldn’t be an issue.
Thank you for your response.

Reply:
@choonho @GibeomOh 
Because we use properties of api response and API has different response spec depends on version, making user to change api version might invoke internal server error(unexpected error). 
So I think we should not give opportunity to change api version. 
Or before change api version, we can add test logic for checking issues.

What do you think about my opinion?

Reply:
@junho100 
That’s a good point.

While it’s challenging to change the content displayed in Cloudforet due to its essential nature, there is a risk of response fields changing with API version updates. Allowing users to change the version might lead to errors.
Having a fixed API version within the connector would help prevent errors and ensure reliable testing.

If a version is supported, the connector can verify which versions are available. 
However, considering all the changes, it might not be necessary to think about it here.
Additionally, it’s unfortunate that there is no NHN Cloud Client.